src/Form/Annotation/AnnotationBuilder.php

```diff
@@ -76,31 +77,31 @@ public function getFormSpecification($entity)
         $metadata     = $this->objectManager->getClassMetadata(is_object($entity) ? get_class($entity) : $entity);
         $inputFilter  = $formSpec['input_filter'];
 
-        $formElements = array(
-            'DoctrineModule\Form\Element\ObjectSelect',
-            'DoctrineModule\Form\Element\ObjectMultiCheckbox',
-            'DoctrineModule\Form\Element\ObjectRadio',
-        );
+        $formElements = [
+            Element\ObjectSelect::class,
```

Comment:
@localheinz in general you have right, it will be better to have everything in separate PR, but then we will have chain of PRs dependent on each other and if something will be missed is some of the beginning PR then will be very difficult to update all of it.

You can try cherry-pick commits from my PR and create separate PRs but I'm not sure if it is worth.